packages/kbn-i18n/babel.config.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
   env: {
     web: {
       presets: ['@kbn/babel-preset/webpack_preset'],
+      plugins: [require.resolve('@babel/plugin-transform-modules-commonjs')],
```

Comment:
A quick hack until we can fix the fact that types are re-exported which is not supported.